pkg/controller/storage/mem/storage.go

```diff
@@ -24,17 +24,17 @@ import (
 type memStorage struct {
 	brokers        *memStorageBroker
 	serviceClasses *memStorageServiceClass
-	instances      map[string]*memStorageInstance
-	bindings       map[string]*memStorageBinding
+	instances      map[string]storage.InstanceStorage
+	bindings       map[string]storage.BindingStorage
```

Comment:
More of a golang question... is `storage.BindingStorage` a pointer since its defined as an interface? Just wondering why we don't have `*` on these.